README.md

```diff
@@ -39,6 +39,16 @@ or
 
 1. ```make up-test-mode```
 
+### How to run ATDD
+
+Steps:
+1. Build Kilda controller. See *"How to Build Kilda Controller"* section.
+2. Run Kilda controller in *"test mode"*. ```make up-test-mode```
+3. Update your /etc/hosts file. Replace ```127.0.0.1 localhost``` to 
+   ```127.0.0.1    localhost nimbus.pendev kafka.pendev logstash neo4j.pendev logstash.pendev```
```

Comment:
`kafka.pendev` is enought to run atdd... But this `requirement` is our tech dept, it must be payed(fixed).

Reply:
ok. I will try with only kafka.pendev host and make changes to the patch.